benchmark/single-source/InsertCharacter.swift

```diff
@@ -0,0 +1,115 @@
+//===--- InsertCharacter.swift ------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2018 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import TestsUtils
+
+public let InsertCharacter = [
+    BenchmarkInfo(name: "InsertCharacterEndIndex", runFunction: run_InsertCharacterEndIndex, tags: [.validation, .api], setUpFunction: buildWorkload),
```

Comment:
Do we want to make this fit in 80 char limit?
```swift
BenchmarkInfo(
  name: "InsertCharacterEndIndex",
  runFunction: run_InsertCharacterEndIndex,
  tags: [.validation, .api],
  setupFunction: buildWorkload
),
```

Reply:
Given that we exceed 80 cols in various other places in the benchmark sources, I'm fine with that.